docs/E-Land_Requirements.rst

```diff
@@ -141,17 +140,22 @@ FUN-MVS-06 - Assets of Energy Conversion
 
 :Priority:  LOW
 
-:Progress: In-progress
+:Progress: Done
 
 :Progress message:
 
-The MVS already covers generic conversion assets. This includes generators, transformers, heat pumps and similar.
+The MVS already covers generic conversion assets. How the generic definition can be applied to the individual assets is explained :ref:`here <energy_conversion`. This includes
+
+* :ref:`Electric transformers <_energyconversion_electric_transformers>`
+* :ref:`Power plants (Condensing power plants and Combined heat and power) <power_plants>`
+* :ref:`Heat pumps and Heating, Ventilation, and Air Conditioning (HVAC) assets <energyconversion_hvac>`
+* :ref:`Electrolyzers <energyconversion_electrolyzers>`
 
```

Comment:
You could also add "Other fuel powered plants"  like diesel generator, biomass plant..